hachoir/parser/program/macho.py

```diff
@@ -100,8 +102,7 @@ def createDescription(self):
             self['ncmds'].value,
             '|'.join(f for f in self.FLAGS if self['flags_' + f.lower()].value))
 
-
```

Comment:
Please fix all PEP8 validation errors - in this repo we are strict about keeping PEP8. @masatake has given you a list of the current errors to fix.